bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResponseQueueReader.java

```diff
@@ -86,9 +88,12 @@ public synchronized Row getNextMergedRow() throws IOException {
     if (queueEntry == null) {
       throw new ScanTimeoutException("Timeout while merging responses.");
     }
+    if (!queueEntry.isCompletionMarker()) {
+      call.request(1);
```

Comment:
this means, when the capacity is N and the response consumption is quick we will have about ~N active requests. Could it be a problem? Also at the end, we will have N -1 requests return no result.